============================= test session starts ==============================
platform linux -- Python 3.10.6, pytest-7.2.0, pluggy-1.0.0
rootdir: /root/projects/qvm
plugins: anyio-3.6.2
collected 55 items / 54 deselected / 1 selected

tst/qvm/integration/bench_test.py 
------------------- Testing --------------

system_name     qubits         status    
ScQ-P10        10             Online    
ScQ-P18        18             Online    
ScQ-P136       136            Online    
Fid of random	nan
.

=============================== warnings summary ===============================
qcs/tst/qvm/integration/bench_test.py::TestQuafuBackendRealMachineQvmFrpV2::test_bench_diff_methods_diff_metric[/root/projects/QASMBench/small//adder_n4/adder_n4.qasm,/root/projects/QASMBench/small//dnn_n2/dnn_n2.qasm-ScQ-P18-4-random-kl]
  /root/anaconda3/envs/qcs-noise/lib/python3.10/site-packages/qiskit/result/mitigation/utils.py:94: RuntimeWarning: invalid value encountered in divide
    vec /= shots

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================= 1 passed, 54 deselected, 1 warning in 14.26s =================
